src/devtools/views/components/ComponentInspector.vue

```diff
@@ -99,18 +99,21 @@ export default {
     },
     openInEditor () {
       const file = this.target.file
-      const src = `fetch('/__open-in-editor?file=${file}').then(response => {
+      // Console display
+      const fileName = file.replace(/\\/g, '\\\\')
+      const src = `fetch('/__open-in-editor?file=${encodeURI(file)}').then(response => {
         if (response.ok) {
-          console.log('File ${file} opened in editor')
+          console.log('File ${fileName} opened in editor')
         } else {
-          const msg = 'Opening component ${file} failed'
+          const msg = 'Opening component ${fileName} failed'
           if (__VUE_DEVTOOLS_TOAST__) {
             __VUE_DEVTOOLS_TOAST__(msg, 'error')
           } else {
             console.log('%c' + msg, 'color:red')
           }
           console.log('Check the setup of your project, see https://github.com/vuejs/vue-devtools/blob/master/docs/open-in-editor.md')
         }
+        console.log('Check the setup of your project, see https://github.com/vuejs/vue-devtools/blob/master/docs/open-in-editor.md')
```

Comment:
Did you put the same console.log on purpose here? In case the response is not ok you'll get 2 same logs in the console.